src/main/scala/com/databricks/labs/overwatch/pipeline/Pipeline.scala

```diff
@@ -138,11 +138,20 @@ class Pipeline(_workspace: Workspace, _database: Database,
    */
   protected def initPipelineRun(): this.type = {
     logger.log(Level.INFO, "INIT: Pipeline Run")
+    val dbMeta = spark.sessionState.catalog.getDatabaseMetadata(config.databaseName)
+    val dbProperties = dbMeta.properties
+    val overwatchSchemaVersion = dbProperties.getOrElse("SCHEMA", "BAD_SCHEMA")
+    if (overwatchSchemaVersion != config.overwatchSchemaVersion) {
+      throw new BadConfigException(s"The overwatch DB Schema version is: $overwatchSchemaVersion but this" +
+        s" version of Overwatch requires ${config.overwatchSchemaVersion}. Upgrade Overwatch Schema to proceed " +
+        s"or drop existing database and allow Overwatch to recreate.")
+    }
+
```

Comment:
moved this from Initializer.scala. I want to be able to instantiate the workspace with a different schema but not run a pipeline.